docs/content/developers/building-contributing.md

```diff
@@ -26,70 +26,86 @@ Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) cre
 
-You do not typically have to install anything else other than the downloaded binary; when you run it it will access any Docker images that it needs.
+    !!!warning "macOS and Unsigned Binaries"
+        macOS doesn’t like these downloaded binaries, so you’ll need to bypass the automatic quarantine to use them:
 
-After you’re done, you can delete your downloaded binary and re-link the original Homebrew one:
+        ```bash
+        xattr -r -d com.apple.quarantine /usr/local/bin/ddev
+        ```
 
-```
-sudo rm /usr/local/bin/ddev
-brew link --force ddev
-```
+        (The binaries on the master branch and the final release binaries _are_ signed.)
 
-### Installing a Downloaded Binary in the `$PATH`
+    After you’re done testing, you can delete your downloaded executable, and re-link the original Homebrew one:
 
-Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
+    ```bash
+    sudo rm /usr/local/bin/ddev
+    brew link --force ddev
+    ```
 
-```
-mkdir -p ~/bin
-export PATH="~/bin:$PATH"
-```
+=== "Installing a Downloaded Binary in the `$PATH`"
 
-Next, unzip the ZIP file you downloaded, make it executable, and move it to a folder in your path. Check with `echo $PATH`:
+    ### Installing a Downloaded Binary in the `$PATH`
 
-```
-unzip ddev.zip
-chmod +x ddev && mv ddev ~/bin
-```
+    Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `$HOME/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
 
-Now, close and reopen your terminal, and verify the replacement worked by running `ddev version`. The output should be something like `DDEV version v1.22.3-39-gfbb878843`, instead of the regular `DDEV version v1.22.3`.
+    ```bash
+    mkdir ~/bin
+    echo 'export PATH="$HOME/bin:$PATH"' >>~/.bashrc
+    # Close and reopen your terminal, and verify that it worked with:
+    echo $PATH
+    ```
 
-You need to run `ddev poweroff` and `ddev start` to download the Docker images that it needs.
+    Next, download the ZIP archive, unzip it, make it executable, and move it to the `~/bin` folder:
 
-After you’re done testing, you can delete your downloaded executable, restart your terminal, and again use the standard DDEV:
+    ```bash
+    wget -O ddev.zip [LINK TO ZIP ARCHIVE]
+    unzip ddev.zip
+    chmod +x ddev && mv ddev ~/bin/ddev
```

Comment:
shouldnt the "macOS and Unsigned Binaries" box be added here as well (in particular in case the other sections would get dropped)?